docs/pages/docs/01-cloud-deployment/best-practices.md

```diff
@@ -8,27 +8,52 @@ permalink: /docs/cloud-deployment/best-practices
 folder: docs
 ---
 
-# GC Deployment Best Pracices
+# GC Deployment Best Practices
 
 In this section we shall describe the recommendations for comput and storage aspects of GC deployments
 
 ## Compute
 
 The optimum machine choice offering a good balance between CPU and memory should be equipped with at least 4 vCPUs and 8 GB of RAM.
 Using machines with extra RAM amount above a 25 GB threshold is not expected to yield significant performance improvements.
```

Comment:
Using machines with _additional RAM beyond a 25 GB threshold_ is not expected to yield significant performance improvements.